pytorch/src/main/java/org/bytedeco/pytorch/presets/torch_cuda.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (C) 2020-2023 Samuel Audet, Eduardo Gonzalez
+ *
+ * Licensed either under the Apache License, Version 2.0, or (at your option)
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation (subject to the "Classpath" exception),
+ * either version 2, or any later version (collectively, the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.gnu.org/licenses/
+ *     http://www.gnu.org/software/classpath/license.html
+ *
+ * or as provided in the LICENSE.txt file that accompanied this code.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bytedeco.pytorch.presets;
+
+import org.bytedeco.javacpp.ClassProperties;
+import org.bytedeco.javacpp.LoadEnabled;
+import org.bytedeco.javacpp.Loader;
+import org.bytedeco.javacpp.annotation.*;
+import org.bytedeco.javacpp.tools.Info;
+import org.bytedeco.javacpp.tools.InfoMap;
+import org.bytedeco.javacpp.tools.InfoMapper;
+
+import java.util.List;
+
+/**
+ * @author Samuel Audet, Hervé Guillemet
+ */
+@Properties(
+    //inherit = torch.class,
```

Comment:
If I remember well, if we inherit, all headers from the main presets will be reparsed, while we need only a small subset here.

Reply:
Yes, Parser parses [`allIncludes` which contains inherited includes](https://github.com/bytedeco/javacpp/blob/b9f91d7f41e99f0f209fbc5ef318ef2217c89dc8/src/main/java/org/bytedeco/javacpp/tools/Parser.java#L4343-L4356). What is confusing ? Isn't it meant to work like that ?

Reply:
No, that's fine, it's just that the only reason you're not using is because it's slow, and what you're doing is adding technical debt by maintaining more lines of code.

Reply:
What about defining a common abstract parent ?

Reply:
No need to do that, see https://github.com/bytedeco/javacpp/pull/689#issuecomment-1589240613.